test/functional/ui/screen.lua

```diff
@@ -205,6 +205,8 @@ end
 
 function Screen:try_resize(columns, rows)
   uimeths.try_resize(columns, rows)
+  -- Give ourselves a chance to _handle_resize
+  self:sleep(0.1)
```

Comment:
Does it allow [this pending test](https://github.com/neovim/neovim/blob/d3b4764dc137e0f6f2c219244e00fba92a64a384/test/functional/ui/screen_basic_spec.lua#L567) to be re-enabled?

Instead of sleeping 100ms, can this use `helpers.retry()` to keep trying every 1ms until `_height` is correct?